watertap/unit_models/electrodialysis_0d.py

```diff
@@ -0,0 +1,938 @@
+        self.scaling_factor = Suffix(direction=Suffix.EXPORT)
+
+        # Next, get the base units of measurement from the property definition
+
+        # Create essential sets.
+        self.membrane_set = Set(initialize=["cem", "aem"])
+
+        # Create unit model parameters and vars
+        self.water_density = Param(
+            initialize=1000,
+            mutable=False,
+            units=pyunits.kg * pyunits.m**-3,
+            doc="density of water",
+        )
+        self.water_MW = Param(
+            initialize=18.015e-3,
+            mutable=False,
+            units=pyunits.kg * pyunits.mole**-1,
+            doc="molecular weight of water",
+        )
+        self.cell_pair_num = Var(
+            initialize=1,
+            domain=NonNegativeIntegers,
+            bounds=(1, Inf),
+            units=pyunits.dimensionless,
+            doc="cell pair number in a stack",
+        )
+
+        # electrodialysis cell dimensional properties
+        self.cell_width = Var(
+            initialize=0.1,
+            bounds=(1e-3, 1e2),
+            units=pyunits.meter,
+            doc="The width of the electrodialysis cell, denoted as b in the model description",
+        )
+        self.cell_length = Var(
+            initialize=0.5,
+            bounds=(1e-3, 1e2),
+            units=pyunits.meter,
+            doc="The length of the electrodialysis cell, denoted as l in the model description",
+        )
+        self.spacer_thickness = Var(
+            initialize=0.0001,
+            units=pyunits.meter,
+            doc="The distance between the concecutive aem and cem",
+        )
+
+        # Material and Operational properties
+        self.membrane_thickness = Var(
+            self.membrane_set,
+            initialize=0.0001,
+            bounds=(1e-6, 1e-1),
+            units=pyunits.meter,
+            doc="Membrane thickness",
+        )
+        self.solute_diffusivity_membrane = Var(
+            self.membrane_set,
+            self.config.property_package.ion_set
+            | self.config.property_package.solute_set,
+            initialize=1e-10,
+            # bounds=(1e-11, 1e-6),
+            units=pyunits.meter**2 * pyunits.second**-1,
+            doc="Solute (ionic and neutral) diffusivity in the membrane phase",
+        )
+        self.ion_trans_number_membrane = Var(
+            self.membrane_set,
+            self.config.property_package.ion_set,
+            bounds=(0, 1),
+            units=pyunits.dimensionless,
+            doc="Ion transference number in the membrane phase",
+        )
+        self.water_trans_number_membrane = Var(
+            self.membrane_set,
+            initialize=5,
+            bounds=(0, 1),
+            units=pyunits.dimensionless,
+            doc="Transference number of water in membranes",
+        )
+        self.water_permeability_membrane = Var(
+            self.membrane_set,
+            initialize=5,
```

Comment:
```suggestion
            initialize=1e-14,
```
We should not be giving an unreasonable default value for any variable.